includes/validation/class-amp-validation-manager.php

```diff
@@ -600,6 +600,14 @@ public static function init_validate_request() {
 		if ( true === $should_validate_response ) {
 			self::add_validation_error_sourcing();
 			self::$is_validate_request = true;
+
+			$display_errors = ini_get( 'display_errors' );
+
+			if ( ! empty( $display_errors ) || 'stderr' !== $display_errors ) {
+				// Suppress the display of fatal errors that may arise during validation so that they will not be counted
+				// as actual validation errors.
+				ini_set( 'display_errors', false );
```

Comment:
WordPress core is using `0` here:

```suggestion
				ini_set( 'display_errors', 0 );
```

Reply:
Done in https://github.com/ampproject/amp-wp/pull/4746/commits/0614c80de60edf4aff4859250ce063fee0874af4.